sdks/python/apache_beam/runners/interactive/background_caching_job.py

```diff
@@ -132,7 +256,22 @@ def is_source_to_cache_changed(user_pipeline):
   is_changed = not current_signature.issubset(recorded_signature)
   # The computation of extract_unbounded_source_signature is expensive, track on
   # change by default.
-  if is_changed:
+  if is_changed and update_cached_source_signature:
+    if ie.current_env().options.enable_capture_replay:
+      if not recorded_signature:
+        _LOGGER.info(
+            'Interactive Beam has detected you have unbounded sources '
+            'in your pipeline. In order to have a deterministic replay '
+            'of your pipeline: {}'.format(
```

Comment:
Half sentence `In order to have a deterministic replay of your pipeline` ... What happens in this case?

Reply:
`ie.current_env().options.capture_control` is formatted the same to the other case.
Do you think we should make its first letter lower case for this scenario?

Reply:
I think both cases should result in a complete sentence.

Reply:
Got it, make it a full sentence.